docs/proposals/PROPOSAL_G.md

```diff
@@ -0,0 +1,63 @@
+# Proposal G - Image Compatibility Artifact
+
+The proposal introduces a new `application/vnd.oci.image-compatibility.v1` artifact type.
+Interest groups/communities SHOULD implement tools to address their specific (or common) use cases.
+
+## Artifact Manifest
+
+An artifact manifest MAY contain multiple blobs with different or the same media types, but MUST contain at least one.
+The new artifact type for image compatibility MUST be referenced in OCI documentation.
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.oci.image-compatibility.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1alpha1+json",
+      "digest": "sha256:4a47f8ae4c713906618413cb9795824d09eeadf948729e213a1ba11a1e31d052",
+      "size": 1710,
+      "annotations": {
+        "name": "profileA"
+      }
+    }, 
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1alpha1+json",
+      "digest": "sha256:35589b7dcd49cbcc48a22815aceccaf34190189a3f25c6e84772771e051a992c",
+      "size": 450,
+      "annotations": {
+        "name": "profileB"
+      }
+    },
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1alpha1+json",
+      "digest": "sha256:494b04517cada9ba77ebdd3a94ad3e0e10a4d000dada8cb0876cd3c5e90fc826",
+      "size": 2040,
+      "annotations": {
+        "name": "profileC"
+      }
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2024-01-02T03:04:05Z"
+  }
+}
+```
+
+## Compatibility Specification
+
+Interest groups/communities are responsible to release their own media types with corresponding schemas and overall JSON structure.
+Schema MUST provide attributes that will allow to express image compatibility.
+Known media types SHOULD be referenced in the OCI documentation.
+
```

Comment:
This looks to be exactly the same as F, but with no suggestion to structure to the artifact itself.

Reply:
So after all it's not the same :)

Reply:
It's just details that are different - the overall media type (for a community) -> artifact is the same I think.

Reply:
Right. But that's a detail I wanted to highlight and that brings a topic to discuss. The artifact indeed is the same.